client/sites/tools/logs/components/site-logs-toolbar/style.scss

```diff
@@ -2,27 +2,68 @@
 @import "@wordpress/base-styles/mixins";
 
 .site-logs-toolbar {
+	align-items: center;
 	display: flex;
 	flex-wrap: wrap;
 	gap: 16px;
 	justify-content: space-between;
 	margin-bottom: 1.5em;
 
+	input:invalid {
+		background-color: var(--color-accent-5);
+		color: var(--color-accent-70);
+		border-color: var(--color-accent-60);
+	}
+
+	input:valid {
+		background-color: var(--color-surface);
+		color: var(--color-neutral-70);
+		border-color: var(--color-neutral-10);
+	}
```

Comment:
These selectors previously lived in the top scope. That's a classic source of bugs for us, so I moved them in here.